rust/telemetry/src/feature_flags.rs

```diff
@@ -0,0 +1,144 @@
+use std::{sync::LazyLock, time::Duration};
+
+use anyhow::{bail, Context as _, Result};
+use parking_lot::RwLock;
+use serde::{Deserialize, Serialize};
+use tokio::runtime::Runtime;
+
+const POSTHOG_API_KEY: &str = "phc_uXXl56plyvIBHj81WwXBLtdPElIRbm7keRTdUCmk8ll";
+const RE_EVAL_DURATION: Duration = Duration::from_secs(5 * 60);
```

Comment:
Maybe a little easier to read?

```suggestion
const EVAL_FLAGS_EVERY: Duration = Duration::from_secs(5 * 60);
```